common/common/src/test/java/io/helidon/build/common/FileUtilsTest.java

```diff
@@ -50,30 +50,25 @@
  */
 public class FileUtilsTest {
 
-    private static Path zipDir;
     private static Path outputDir;
-    private static Path zipFile;
 
     @BeforeAll
     static void setup() throws IOException {
         outputDir = Files.createDirectories(
-                TestFiles.targetDir(FileUtilsTest.class).resolve("test-classes/archives"));
-        zipFile = createZip();
-        zipDir = Files.createTempDirectory("zip");
-        Files.createDirectories(zipDir.resolve("dir"));
-        Files.createFile(zipDir.resolve("file1"));
-        Files.createFile(zipDir.resolve("dir/file2"));
+                TestFiles.targetDir(FileUtilsTest.class).resolve("test-classes/file-utils"));
     }
 
     @Test
     @EnabledOnOs({OS.LINUX, OS.MAC})
-    void testZipPermissions() {
+    void testZipPermissions() throws IOException {
+        Path wd = outputDir.resolve("zip-permissions");
```

Comment:
What happens if you re-run the test ?
You should use `FileUtils.unique` to ensure that the file or directory does not exist.
Since we create all these files and directories under `target` they will be cleaned-up with the next `mvn clean`.